src/components/UI/search-box/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { searchMemberContext } from '@store/search-members/searchMembers-context';
+import classNames from './search-box.module.scss';
+
+const SearchBox = () => {
+  const { setSearchTerm } = searchMemberContext();
+  return (
+    <div className={classNames.searchContainer}>
+      <input
+        type="text"
+        placeholder="&#128269; search members here"
```

Comment:
Can we please get this value from constants?
I think we have a constants file - https://github.com/Real-Dev-Squad/website-members/blob/develop/src/constants/AppConstants.js

Reply:
Done. Please check